newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -209,18 +242,22 @@ const selectLocalResourcePath = (
     if (options.multiSelections) properties.push('multiSelections');
     const projectPath = path.dirname(project.getProjectFile());
 
+    const latestPath = getLastUsedPath(project, kind) || projectPath;
+
     const browserWindow = electron.remote.getCurrentWindow();
     dialog.showOpenDialog(
       browserWindow,
       {
         title: options.title,
         properties,
         filters: [{ name: options.name, extensions: options.extensions }],
-        defaultPath: projectPath,
+        defaultPath: latestPath,
       },
       paths => {
         if (!paths) return resolve([]);
 
+        setLastUsedPath(project, kind, paths[0]);
```

Comment:
Do you want to save the full path, or the directory name only?

Reply:
@4ian Currently, I am saving full path.

Reply:
Also, it may happen that the last path for any resource (eg. audio), maybe an altogether different directory or different drive partition. So saving only the directory name will not work.

Reply:
I mean in the case of /myfolder/subfolder/myfile.txt, to save /myfolder/subfolder/
Which means on Windows, instead of C:\something\game.json, saving C:\something

after all, we don't care about which file you last used right, just which folder you used? :)

Reply:
Oh! I see. I will update it :)